import-secrets.sh

```diff
@@ -3,6 +3,11 @@ set -e
 
 AWS_CREDS_ROLE="${AWS_CREDS_ROLE:-epoch-fleet-manager}"
 
+# Vault address secret used by Terraform, because it cannot be sources in TF
+if [ -n "$VAULT_ADDR" ]; then
+    export TF_VAR_vault_addr=$VAULT_ADDR
```

Comment:
Unused var?

Reply:
It's [Terraform variable](https://github.com/aeternity/infrastructure/pull/132/files#diff-b89d1f9fe4d022751a376ef53e46021dR9) set as [environment variable](https://www.terraform.io/docs/configuration/variables.html#environment-variables).